### DISCUSSION TITLE
SSL Cert not applying to my subdomain

### DISCUSSION THREAD

**Discussion by catedm** [vercel/vercel#6027] [Help]
I added a subdomain, and the ssl cert installed by Vercel is not applying to the subdomain. I see the wildcard CN for the subdomain in the SSL Certificates section, but the cert is not working on my subdomain. What am I missing?

**Comment by fu-sen** [comment]
https://vercel.com/docs/custom-domains#wildcard-domains

> WARNING: If using your Custom Domain as a Wildcard Domain, you must use the nameservers method for verification.

Is the name server set up properly? It is mandatory if Let's Encrypt issues a wildcard certificate.

**Reply by catedm** [reply]
I have a top level domain set up with the proper nameserver configuration. SSL is working for that domain (`example.com`)

I added a subdomain and pointed it to an IP address on an instance that I have running in AWS. SSL cert did not work for the subdomain.

Are you saying that I need to add my top level domain in Vercel as `*.example.com`, instead of `example.com`?

**Reply by fu-sen** [reply]
> I added a subdomain and pointed it to an IP address on an instance that I have running in AWS. SSL cert did not work for the subdomain.

The explanation was lacking.
So does that mean that your subdomain is on AWS and you want to issue a certificate there?

Is the name server for that domain managed by Vercel?
The Vercel name server has the following records:
```
CAA | 0 issue "letsencrypt.org"
```

AWS issues different certificates. It may be solved by adding a CAA record.

**Reply by catedm** [reply]
I added the subdomain in Vercel using an `A` record. I pointed the `A` record to the IP address attached to my AWS instance.

<img width="1010" alt="Screen Shot 2021-03-25 at 10 10 07 AM" src="https://user-images.githubusercontent.com/19396301/112486630-7124bd80-8d52-11eb-9000-0c2674665c06.png">

The subdomain successfully points to my AWS instance. But the SSL cert is not applying there. Do I need to add the SSL cert to the AWS instance?

**Reply by fu-sen** [reply]
No progress! Do you understand the meaning of CAA records?

**Reply by catedm** [reply]
The nameserver is managed by Vercel. I added a subdomain using an `A` record and pointed it to the IP address of my AWS instance.

I am not sure exactly what CAA records are.
